script-library/container-features/src/features.json

```
@@ -965,7 +965,7 @@
 					"type": "string",
 					"enum": ["latest"],
```

Since we can fall back to pulling from a repository here we likely can safely allow people to enter a version.  So this could be a `proposal`.  7.2 is coming out soon, so listing latest and 7.1 is probably good.
```suggestion
					"proposals": ["latest", "7.1"],
```